sources/packages/backend/apps/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -305,7 +310,7 @@ export class ApplicationStudentsController extends BaseController {
   @ApiOkResponse({ description: "Draft application updated." })
```

Comment:
Same here https://github.com/bcgov/SIMS/pull/1636/files#r1063629217. In this case it will not show the request body schema in swagger.